docs/spark/tutorials/databricks-deployment.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Deploy a .NET for Apache Spark application to Databricks
+description: Discover how to deploy a .NET for Apache Spark application to Databricks.
+ms.date: 05/14/2019
+ms.topic: tutorial
+ms.custom: mvc
+#Customer intent: As a developer, I want to deployment .NET for Apache Spark application to Databricks.
+---
+
+# Deploy a .NET for Apache Spark application to Databricks
+
+This tutorial teaches how to deploy a .NET for Apache Spark application to Databricks.
```

Comment:
It is Azure Databricks, or AWS Databricks or both?

Reply:
it's both... that's why I just put Databricks. I mention and link to both later in the article. Do you think I should separate them here, too?